drivers/dma/Kconfig.stm32

```
@@ -25,9 +26,16 @@ config DMA_STM32_V1
 
 config DMA_STM32_V2
 	bool
-	default y if $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_DMA_V2))
+	default y if $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_DMA_V2)) \
+			|| $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_DMA_V2BIS))
 	help
 	  Enable DMA V2 support.
+	  "DMA_STM32_V2 symbol controls code that is common to both st,stm32-dma
+	  and st,stm32-dma-v2bis compatible. Behavior difference between these bindings
+	  is that the st,stm32-dma-v2bis has all the requests from the peripherals
+	  logically ORed to one input request, like in the stm32F1 or stm32L1.
+	  DMA_STM32_V2BIS is enabled using compatible specific macros.
+	  Note that a DMA of type V2bis has a same driver code as V2.
```

```suggestion
	  DMA_STM32_V2 symbol controls code that is common to both "st,stm32-dma"
	  and "st,stm32-dma-v2bis" compatibles. Behavior difference between these bindings
	  is that the "st,stm32-dma-v2bis" has all the requests from the peripherals
	  logically ORed to one input request, like in the STM32F1 or STM32L1.
	  Note that a DMA of type V2bis has a same driver code as V2, and
	  V2bis specific code is controller using compatible specific macros.
```

Refering to DMA_STM32_V2BIS is misleading as this Kconfig symbol does not exist

---

ok